js/background.js

```diff
@@ -566,139 +590,145 @@ var count_cache = (function(count_map) {
     if (count_cache.getCustomFilterCount(host)) {
         remove_custom_filter(host);
         count_cache.removeCustomFilterCount(host);
     }
-}
+};
 
-confirm_removal_of_custom_filters_on_host = function(host, activeTab) {
+var confirm_removal_of_custom_filters_on_host = function(host, activeTab) {
     var custom_filter_count = count_cache.getCustomFilterCount(host);
-    var confirmation_text   = translate("confirm_undo_custom_filters", [custom_filter_count, host]);
+    var confirmation_text = translate("confirm_undo_custom_filters", [custom_filter_count, host]);
     if (!FIREFOX) { // Firefox doesn't support confirm() in BG page yet
-        if (!window.confirm(confirmation_text)) { return; }
+        if (!window.confirm(confirmation_text)) {
+            return;
+        }
     }
     remove_custom_filter_for_host(host);
     if (!SAFARI) {
         chrome.tabs.update(activeTab.id, {url: activeTab.url});
     } else {
-        activeTab.url = activeTab.url
+        activeTab.url = activeTab.url;
     }
 };
 
-get_settings = function() {
+var get_settings = function() {
     return _settings.get_all();
-}
+};
 
-set_setting = function(name, is_enabled) {
+var set_setting = function(name, is_enabled) {
     _settings.set(name, is_enabled);
 
-    if (name === "debug_logging")
+    if (name === "debug_logging") {
         logging(is_enabled);
-}
+    }
+};
 
-disable_setting = function(name) {
+var disable_setting = function(name) {
     _settings.set(name, false);
-}
+};
 
 // MYFILTERS PASSTHROUGHS
 
 // Rebuild the filterset based on the current settings and subscriptions.
-update_filters = function() {
+var update_filters = function() {
     _myfilters.rebuild();
-}
+};
 
 // Fetch the latest version of all subscribed lists now.
-update_subscriptions_now = function() {
+var update_subscriptions_now = function() {
     _myfilters.checkFilterUpdates(true);
-}
+};
 
 // Returns map from id to subscription object.  See filters.js for
 // description of subscription object.
-get_subscriptions_minus_text = function() {
+var get_subscriptions_minus_text = function() {
     var result = {};
     for (var id in _myfilters._subscriptions) {
         result[id] = {};
         for (var attr in _myfilters._subscriptions[id]) {
-            if ((attr === "text") || (attr === "rules")) continue;
+            if ((attr === "text") || (attr === "rules")) {
+                continue;
+            }
             result[id][attr] = _myfilters._subscriptions[id][attr];
         }
     }
     return result;
-}
+};
 
 // Get subscribed filter lists
-get_subscribed_filter_lists = function() {
+var get_subscribed_filter_lists = function() {
     var subs = get_subscriptions_minus_text();
     var subscribed_filter_names = [];
     for (var id in subs) {
-        if (subs[id].subscribed)
+        if (subs[id].subscribed) {
             subscribed_filter_names.push(id);
+        }
     }
     return subscribed_filter_names;
-}
+};
```

Comment:
Useless semicolon. Braces indicating end of multi-line statement here, so semicolon indicating end of single-line statement not needed.